Adafruit_CAP1188.cpp

```diff
@@ -102,105 +152,113 @@ boolean Adafruit_CAP1188::begin(uint8_t i2caddr) {
   }
 
   readRegister(CAP1188_PRODID);
-  
+
   // Useful debugging info
-  
+
   Serial.print("Product ID: 0x");
   Serial.println(readRegister(CAP1188_PRODID), HEX);
   Serial.print("Manuf. ID: 0x");
   Serial.println(readRegister(CAP1188_MANUID), HEX);
   Serial.print("Revision: 0x");
   Serial.println(readRegister(CAP1188_REV), HEX);
-  
 
-  if ( (readRegister(CAP1188_PRODID) != 0x50) ||
-       (readRegister(CAP1188_MANUID) != 0x5D) ||
-       (readRegister(CAP1188_REV) != 0x83)) {
+  if ((readRegister(CAP1188_PRODID) != 0x50) ||
+      (readRegister(CAP1188_MANUID) != 0x5D) ||
+      (readRegister(CAP1188_REV) != 0x83)) {
     return false;
   }
   // allow multiple touches
-  writeRegister(CAP1188_MTBLK, 0); 
+  writeRegister(CAP1188_MTBLK, 0);
   // Have LEDs follow touches
   writeRegister(CAP1188_LEDLINK, 0xFF);
   // speed up a bit
   writeRegister(CAP1188_STANDBYCFG, 0x30);
   return true;
 }
 
-uint8_t  Adafruit_CAP1188::touched(void) {
+/*!
+ *   @brief  Reads from CAP1188_SENINPUTSTATUS and if touch is detected writes to CAP1188_MAIN
+ *   @return Returns read from CAP1188_SENINPUTSTATUS
+ */
+uint8_t Adafruit_CAP1188::touched() {
```

Comment:
this function reads the touched status, returns an 8-bit value where the 0 pad is the LSB (1 if touched, 0 if not)